Automattic/jetpack
# Nav Unification: Use correct user capability for the Inbox menu item

Related to #19506 (see https://github.com/Automattic/jetpack/issues/19506#issuecomment-967298882)

#### Changes proposed in this Pull Request:

* Use correct user capability (`manage_options`) for the Inbox menu item in Nav Unification.

The Inbox screen in Calypso is [limited to users that can `manage_options`](https://github.com/Automattic/wp-calypso/blob/4cbfaeaa41e59d2579ba27e6588495d1920a3b4a/client/my-sites/email/inbox/index.jsx#L72).
Though, in the Nav Unification, the Inbox menu item was added with the `edit_posts` capability, which is very low in the permissions hierarchy.

This resulted in displaying the Inbox menu item to users with pretty much any role, but clicking on it would show them an unauthorized message.

| Admin | Contributor |
| - | - |
| <img width="269" alt="Screenshot 2021-11-26 at 14 11 11" src="https://user-images.githubusercontent.com/2070010/143593653-ee21fac6-4679-46f0-b741-a517eeef672f.png"> | <img width="265" alt="Screenshot 2021-11-26 at 14 11 45" src="https://user-images.githubusercontent.com/2070010/143593672-82564905-96ed-4e25-8cb3-7397009da68e.png"> |

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Prepare WPCOM to test.
  * Sync these changes to WPCOM.
  * Sandbox the API.
  * Sandbox a WPCOM site with two test users, an Admin and a Contributor.
* Open that site in wp-admin.
  * With the Admin user: make sure the Inbox menu is visible.
  * With the Contributor user: make sure the Inbox menu is not visible
* Repeat the tests in Calypso.
  * With the Admin user: make sure the Inbox menu is visible.
  * With the Contributor user: make sure the Inbox menu is not visible
